mysql-test/suite/innodb/t/xtradb_compressed_columns_upgrade-master.opt

```diff
@@ -0,0 +1 @@
+--loose-compressed-columns-unknown1
```

Comment:
The idea behind this was to create a unique set of options for this test case (which in turn do nothing) so that it would always start from a fresh copy of system tables. This is needed to make sure that MAX tablespace ID and max index ID are still lower than those in pre-included 5.6 .ibd files (100 and 150 in our case). Without this hack whether this test is successful or not will depend on how many tests MTR has run before executing this one.